### PR TITLE
Implemented add terms portion of Manage Taxonomy drawer

### PR DESCRIPTION
**Pull request by noisecapella**

### CODE REVIEW

File: .jscsrc [hunk 1]
```diff
@@ -1,3 +1,4 @@
 {
-  "preset": "google"
+  "preset": "google",
+  "validateQuoteMarks": null
```
Comment (carsongee) [inline .jscsrc:3]: hehe, that was bugging me too, thanks

**Comment by carsongee** [comment]
I'd like to add tests to this, but I'll add that as an issue.  This has merge conflicts or I'd merge, can you rebase?

#249

**Comment by pdpinch** [comment]
@pwilkins can you take a look at this as well?

**Comment by carsongee** [comment]
This can be a sep issue, but it would be nice the enter key had the same action as clicking the plus button

#272

**Comment by noisecapella** [comment]
I'm fine making the enter key action a separate issue

**Comment by pwilkins** [comment]
I was able to add terms to a vocabulary, and verified that they were added by querying the API:  
`{{base-url}}/repositories/animating-stuffed-animals/vocabularies/dimensionality/terms`

**Comment by pdpinch** [comment]
I opened new issues so I believe this can be merged. (The coverage failure seems trivial to me?)

**Comment by pwilkins** [comment]
@noisecapella @carsongee  It appears that this is failing coveralls because line 55 in  /importer/api/**init**.py  is not tested.
https://coveralls.io/builds/2943098/source?filename=importer%2Fapi%2F__init__.py#L55
Given its trivial nature, could we just add `pylint: no coverage`?

**Comment by pwilkins** [comment]
Once tests are fixed, I can merge.

**Pull request merged**